lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,155 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # Prevent autorun from running tests when the VM closes
+  #
+  # Mutant needs control about the exit status of the VM and
+  # the moment of test execution
+  #
+  # @api private
+  #
+  # @return [nil]
+  def self.autorun
+  end
+
+end # Minitest
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      TEST_METHOD_PREFIX = 'test_'.freeze
+      TEST_FILE_PATTERN  = './test/**/{test_*,*_test}.rb'.freeze
```

Comment:
I pulled down the 100 most popular gems depending on minitest to see if the convention was this clear cut. A few differences I noticed: 

* The [json gem](https://github.com/flori/json) which is now part of ruby's standard lib uses a "tests" directory which does not match this glob
* The [excon](https://github.com/excon/excon) gem uses a "tests" directory then I think their test files have a `_tests.rb` suffix

Still looking for other deviations right now so I'll probably update this comment.

My conclusion here isn't necessarily that the glob should change though. Instead, I think this indicates that it might be important for to have a `--test-pattern` flag in the future which makes this configurable.